examples/enclave_manager/setup.py

```diff
@@ -61,57 +57,34 @@
 
 include_dirs = [
     module_src_path,
+    enclave_bridge_path,
     os.path.join(tcf_root_dir, 'tc/sgx/common'),
     os.path.join(tcf_root_dir, 'tc/sgx/common/crypto'),
     os.path.join(module_src_path, 'build'),
```

Comment:
I think we can remove the build folder from include path. Please verify

Reply:
Done